packages/discord.js/library/use-reactions.ts

```diff
@@ -0,0 +1,50 @@
+import {
+	Collection,
+	type Message,
+	type MessageReaction,
+	type ReactionCollector,
+} from "discord.js"
+import { useCallback, useEffect, useRef, useState } from "react"
+import { useMessage } from "./use-message"
+
+interface Options {
+	message?: Message
+}
+
+type Reactions = Collection<string, MessageReaction>
+
+export function useReactions({ message }: Options) {
+	// Hooks should not be called conditionally
```

Comment:
This comment should point out the reason I'm calling the hook non-conditionally because of rules of hooks. Just in case someone opens an issue or decides to make a change